tests/components/geofency/test_init.py

```diff
@@ -104,6 +104,12 @@
 }
 
 
+@pytest.fixture(autouse=True)
+def mock_dev_track(mock_device_tracker_conf):
+    """Mock device tracker config loading."""
+    pass
+
+
 @pytest.fixture
 def geofency_client(loop, hass, hass_client):
```

Comment:
Is the new fixture used for other fixtures in this module too? If we activate it for this fixture it looks like we don't need to mock `'homeassistant.components.device_tracker.update_config'` inside here anymore.